core-bundle/contao/templates/twig/content_element/swiper.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends "@Contao/content_element/_base.html.twig" %}
+{% use "@Contao/component/_stylesheet.html.twig" %}
+
+{% block content %}
+    <div class="swiper" data-delay="{{ delay }}" data-speed="{{ speed }}" data-offset="{{ offset }}" data-loop="{{ loop }}">
+        <div class="swiper-wrapper">
+            {% for element in nested_fragments %}
+                <div class="swiper-slide">
+                    {{ content_element(element) }}
+                </div>
+            {% endfor %}
+        </div>
+        <div class="swiper-button-next"></div>
+        <div class="swiper-button-prev"></div>
+        <div class="swiper-pagination"></div>
+    </div>
+{% endblock %}
+
+{% block script %}
+    {% add "swiper_js" to body %}
+        <script src="{{ asset('js/swiper-bundle.min.js', 'contao-components/swiper') }}"></script>
+        <script>
+            (function() {
+                const swiper = document.querySelectorAll('.swiper');
+                swiper.forEach (el => {
+                    let opts = {
+                        speed: el.getAttribute('data-speed'),
+                        initialSlide: el.getAttribute('data-offset'),
+                        loop: el.getAttribute('data-loop'),
```

Comment:
```suggestion
                        loop: el.hasAttribute('data-loop'),
```
Needs to be changed if we want to use `.set('data-loop', loop)` as suggested in the other comment.

Reply:
See fdf68f42d64dcfbb1b6422ebe68ffdc9ffbb3886.